src/api/internal.rs

```diff
@@ -265,24 +268,25 @@ impl<T: Pixel> ContextInner<T> {
       if enc.quantizer < 255 { Some(enc.quantizer as u8) } else { None };
 
     let seq = Sequence::new(enc);
+    let inter_cfg = InterConfig::new(enc);
+    let lookahead_distance = inter_cfg.keyframe_lookahead_distance() as usize;
     ContextInner {
       frame_count: 0,
       limit: None,
-      inter_cfg: InterConfig::new(enc),
+      inter_cfg,
       output_frameno: 0,
       frames_processed: 0,
       frame_q: BTreeMap::new(),
       frame_data: BTreeMap::new(),
-      keyframes: BTreeSet::new(),
+      keyframes: [0].iter().cloned().collect(),
```

Comment:
How is this a simplification?

Reply:
it avoids a branch in the send_frame(). I can make setting BTreeSet in two lines that are less surprising.